modules/core/infrastructure/persistence/mappers.go

```diff
@@ -16,11 +16,11 @@ import (
 	"github.com/iota-agency/iota-sdk/pkg/domain/entities/upload"
 )
 
-func ToDomainUser(dbUser *models.User) *user.User {
+func ToDomainUser(dbUser *models.User) (*user.User, error) {
 	roles := make([]*role.Role, len(dbUser.Roles))
-	for i, r := range dbUser.Roles {
-		roles[i] = toDomainRole(&r)
-	}
+	// for i, r := range dbUser.Roles {
```

Comment:
Did you meant to leave this?

Reply:
No, but I will clean it up later when it's ready to merge